src/Bicep.Core/Syntax/ModuleDeclarationSyntax.cs

```diff
@@ -37,14 +36,12 @@ public ModuleDeclarationSyntax(Token keyword, IdentifierSyntax name, SyntaxBase
         public SyntaxBase Path { get; }
 
         public SyntaxBase Assignment { get; }
-
-        public SyntaxBase? IfCondition { get; }
-
-        public SyntaxBase Body { get; }
+        
+        public SyntaxBase Value { get; }
```

Comment:
>Value [](start = 26, length = 5)

I initially change the name to break all the references to this property to force myself to inspect every usage of it. I think renaming back to `Body` would be misleading, but open to other names.